tests/rptest/tests/shard_placement_test.py

```diff
@@ -586,7 +586,10 @@ def node_rebalance_finished():
 
             return True
 
-        wait_until(node_rebalance_finished, timeout_sec=60, backoff_sec=2)
+        wait_until(node_rebalance_finished,
```

Comment:
would it be possible to check if the rebalance process makes progress instead of increasing timeout

Reply:
done